### ISSUE TITLE
zTranslate is being ignored selectively by Blink

### ISSUE THREAD

**Issue by StephanBijzitter** [Famous/famous#522]
demo: http://skelware.com/famous-material/demo/3/
source: https://github.com/StephanBijzitter/Famous-Material-Examples/tree/demo-3
Famous version: 0.3.1 (installed with bower)
Screenshot: http://puu.sh/drw62.png

The horizontal white backgrounds have a zTranslate of 5 while the content that's (supposed to be) on top of them have a zTranslate of 16. Yet, the background still manages to be on top.

======
# TLDR: Blink ignores z-order of a child when its parent's overflow is hidden or when ANY of its parents' transform style is flat.

**Comment by MylesBorins** [comment]
We are no longer actively maintaining this repo and as such we will not be fixing this bug.

Thanks you for taking the time to discuss the issue, we hope you will bring the same drive an  spirit to the famous engine, which can be found at --> http://github.com/famous/engine
